Documentation/packaging.md

```diff
@@ -81,6 +60,34 @@ Packages using the `--with-stage1-flavors=host` option must therefore add a run-
 Whenever systemd and bash are upgraded on the host, rkt will use the new version at run time.
 It becomes the packager's responsibility to test the rkt package whenever a new version of systemd is packaged.
 
-# Other code bundle
+### Other build considerations
+
+#### Godep
+rkt uses [Godep](https://github.com/tools/godep) to maintain [a copy of dependencies in its source repository](https://github.com/coreos/rkt/tree/master/Godeps).
+
+## Run-time dependencies
+
+A Linux system configured with [suitable options](hacking.md#run-time-requirements) is required.
+
+### Additional run-time dependencies for the host flavor
+
+- systemd >= v222 with systemctl, systemd-shutdown, systemd, systemd-journald.
+- bash
+
+### Ownership and permissions of rkt directories
```

Comment:
"Ownership and permissions of rkt directories" should have 2 `#` so that it is not in "Run-time dependencies".